src/locales/pt_BR/battle.ts

```diff
@@ -25,6 +25,7 @@ export const battle: SimpleTranslationEntries = {
   "hitResultOneHitKO": "Foi um nocaute de um golpe!",
   "attackFailed": "Mas falhou!",
   "attackHitsCount": "Acertou {{count}} vezes.",
+  "rewardGain": "You received\n{{modifierName}}!",
```

Comment:
Suggested portuguese translation
```suggestion
  "rewardGain": "Você recebeu\n{{modifierName}}!",
```